website/client-old/js/controllers/authCtrl.js

```diff
@@ -11,6 +11,7 @@ angular.module('habitrpg')
 
       $scope.logout = function() {
         localStorage.clear();
+        window.document.cookie.split(";").forEach(function(c) { window.document.cookie = c.replace(/^ +/, "").replace(/=.*/, "=;expires=" + new Date().toUTCString() + ";path=/"); });
```

Comment:
@crookedneighbor Can't find where I found it, but looking at it again, it does seem off. I will look and see if I can find a different one.